core/arch/arm/include/arm64.h

```diff
@@ -249,6 +249,11 @@ static inline void tlbi_vaae1is(uint64_t mva)
 	asm volatile ("tlbi	vaae1is, %0" : : "r" (mva));
 }
 
+static inline void tlbi_vale1is(uint64_t mva)
+{
+	asm volatile ("tlbi	vaae1is, %0" : : "r" (mva));
```

Comment:
The name of the function is `*val*` but here we have `vaa`, why? Spelling mistake?

Reply:
Well spotted, it's a spelling mistake. I'll fix.